eng/update-band.ps1

```diff
@@ -1,25 +1,41 @@
 [CmdletBinding(PositionalBinding=$false)]
 Param(
-    [Parameter(Mandatory = $true)]
+    [Parameter(Mandatory = $false)]
     [string]$Remote,
 
-    [Parameter(Mandatory = $true)]
-    [Alias("branch-1xx")]
+    [Parameter(Mandatory = $false)]
     [string]$Branch1xx,
 
+    [switch]$Continue = $false,
+
     [switch][Alias('h')]$help
 )
 
 function Get-Usage {
     Write-Host "  -Remote <name>           Git remote to pull from (e.g., upstream)"
-    Write-Host "  -1xx-branch <branch>     1xx branch name to merge in (e.g., main)"
+    Write-Host "  -Branch1xx <branch>     1xx branch name to merge in (e.g., main)"
+    Write-Host "  -Continue                Continue after manually fixing merge conflicts"
```

Comment:
Consider a different name. `Continue` is a reserved word so this may cause confusion: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_continue